src/wpool_queue_manager.erl

```diff
@@ -182,7 +200,7 @@ stats(PoolName) ->
 -spec init(wpool:name()) -> {ok, state()}.
 init(WPool) ->
   put(pending_tasks, 0),
-  {ok, #state{wpool = WPool, clients = queue:new(), workers = gb_sets:new()}}.
+  {ok, #state{wpool = WPool, clients = queue:new(), workers = gb_sets:new(), monitors = gb_trees:new()}}.
```

Comment:
Line 203 is too long:   {ok, #state{wpool = WPool, clients = queue:new(), workers = gb_sets:new(), monitors = gb_trees:new()}}..